### ISSUE TITLE
"Search in results" error

### ISSUE THREAD

**Issue by AlexeyOplachko** [sipcapture/homer-app#291]
When "search in results" loads it throws this error in console and also has same behavior as #290 
```
this.initSearchSlider TypeError: "t is null"
    n http://localhost:9080/main.372dc859ca15d227b61d.js:1
    initSearchSlider http://localhost:9080/main.372dc859ca15d227b61d.js:1
    c http://localhost:9080/polyfills.635d703039731e52383d.js:1
    c http://localhost:9080/polyfills.635d703039731e52383d.js:1
    i http://localhost:9080/polyfills.635d703039731e52383d.js:1
    invoke http://localhost:9080/polyfills.635d703039731e52383d.js:1
    onInvoke http://localhost:9080/main.372dc859ca15d227b61d.js:1
    invoke http://localhost:9080/polyfills.635d703039731e52383d.js:1
    run http://localhost:9080/polyfills.635d703039731e52383d.js:1
    O http://localhost:9080/polyfills.635d703039731e52383d.js:1
    invokeTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    onInvokeTask http://localhost:9080/main.372dc859ca15d227b61d.js:1
    invokeTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    runTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    y http://localhost:9080/polyfills.635d703039731e52383d.js:1
    invokeTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    f http://localhost:9080/polyfills.635d703039731e52383d.js:1
    p http://localhost:9080/polyfills.635d703039731e52383d.js:1
main.372dc859ca15d227b61d.js:1:3906029
    initSearchSlider http://localhost:9080/main.372dc859ca15d227b61d.js:1
    c http://localhost:9080/polyfills.635d703039731e52383d.js:1
    c http://localhost:9080/polyfills.635d703039731e52383d.js:1
    i http://localhost:9080/polyfills.635d703039731e52383d.js:1
    invoke http://localhost:9080/polyfills.635d703039731e52383d.js:1
    onInvoke http://localhost:9080/main.372dc859ca15d227b61d.js:1
    invoke http://localhost:9080/polyfills.635d703039731e52383d.js:1
    run http://localhost:9080/polyfills.635d703039731e52383d.js:1
    O http://localhost:9080/polyfills.635d703039731e52383d.js:1
    invokeTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    onInvokeTask http://localhost:9080/main.372dc859ca15d227b61d.js:1
    invokeTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    runTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    y http://localhost:9080/polyfills.635d703039731e52383d.js:1
    invokeTask http://localhost:9080/polyfills.635d703039731e52383d.js:1
    f http://localhost:9080/polyfills.635d703039731e52383d.js:1
    p http://localhost:9080/polyfills.635d703039731e52383d.js:1```

**Comment by RFbkak37y3kIY** [comment]
fixed on devel, waiting merge to master
https://github.com/sipcapture/homer-ui/commit/a19d7875099cd2ce0c334c2041484942521eeaad

**Comment by adubovikov** [comment]
fixed in the UI 7.7.025
